reflex/utils/types.py

```diff
@@ -182,6 +182,26 @@ def is_generic_alias(cls: GenericType) -> bool:
     return isinstance(cls, GenericAliasTypes)
 
 
+def unionize(*args: GenericType) -> Type:
+    """Unionize the types.
+
+    Args:
+        args: The types to unionize.
+
+    Returns:
+        The unionized types.
+    """
+    if not args:
+        return Any
+    if len(args) == 1:
+        return args[0]
+    # We are bisecting the args list here to avoid hitting the recursion limit
+    # In Python versions >= 3.11, we can simply do `return Union[*args]`
```

Comment:
Can we add a case for >=python3.11 here to avoid unneeded computation?

Reply:
turns out it fails with syntax error, meaning we can't have it in the code till 3.11 :L